.env.localci

```
@@ -0,0 +1,233 @@
+#MUNICIPAL_TENANT_PROTOCOL=https://
+#
+PASSPORT_CLIENT_ID=1
+PASSPORT_CLIENT_SECRET=1111111111111111111111111111111111111111
+#
+#MUNICIPAL_SERVER=new1.munworqc.net
+#MUNICIPAL_SERVER=test.packager-tester.test
+#MUNICIPAL_SERVER_LOGIN=test@processoft.com
+MUNICIPAL_SERVER_PASSWORD=secret
+#MUNICIPAL_SERVER_PASSWORD=##password##
+##used in request to allow Cross-origin access, maybe more
+MUNICIPAL_WEBMOBILE_PROXY_SERVER="{$APP_URL}"
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The MUNICIPAL_WEBMOBILE_PROXY_SERVER key is not assigned properly

(SubstitutionKey)